requirements/base.txt

```diff
@@ -8,9 +8,17 @@ django-secure==1.0.1
 # Time zones support
 pytz==2015.4
 
+# mongo
+mongoengine==0.10.0
+
 # API
 djangorestframework==3.2.2
 django-rest-swagger==0.3.4
 drf-nested-routers==0.9.0
 
-requests==2.7.0
\ No newline at end of file
+requests[security]==2.7.0
```

Comment:
This SO: http://stackoverflow.com/a/29099439 suggests that requests[security] is unnecessary on python 2.7.9+. Using it introduces some external apt requirements that are tricky to install in a slug. It's probably a lot simpler to drop python-2.7.10 in a runtime.txt as per https://github.com/heroku/heroku-buildpack-python#specify-a-runtime